src/lib/classes/proxy-deployer.spec.ts

```diff
@@ -23,16 +24,10 @@ describe('UniversalProfile', () => {
     );
     await erc725AccountProxy.initialize(await signer.getAddress());
 
-    await erc725AccountProxy.setData(
-      '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5',
-      '0xbeefbeef',
-      {
-        from: await signer.getAddress(),
-      }
-    );
-    const data = await erc725AccountProxy.getData(
-      '0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5'
-    );
+    await erc725AccountProxy.setData(ERC725YDataKeys.LSP3.LSP3Profile, '0xbeefbeef', {
+      from: await signer.getAddress(),
+    });
+    const data = await erc725AccountProxy.getData(ERC725YDataKeys.LSP3.LSP3Profile);
 
     expect(data).toEqual('0xbeefbeef');
   });
```

Comment:
Or actually I just see here it s equal to the lsp3 metadata so maybe we could put the constant in a variable and name it `expectedLsp3ProfileData` What do you think ?